source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/AdaptiveCardRenderer.java

```diff
@@ -244,7 +246,17 @@ else if (alignment == ActionAlignment.Center.swigValue())
 
         if (viewGroup != null)
         {
-            viewGroup.addView(actionButtonsLayout);
+            if(actionButtonsLayoutOrientation == ActionsOrientation.Horizontal.swigValue())
+            {
```

Comment:
Also this does not work on my machine . The show card action code : 
` 
            ViewGroup parent = (ViewGroup) viewGroup.getParent().getParent(); 
            ViewGroup hiddenCards = (ViewGroup) parent.getChildAt(1);
            hiddenCards.addView(invisibleCard);
`

Looks for the parents's parent first child ( Expected to be show card layout ) . In this case horizontal scroll view is direct parent whose parent is the card layout ( not the root card layout which is expected ) . Hence any action which is show card will not render here